BUILD.gn

```diff
@@ -0,0 +1,46 @@
+# Copyright 2014 The Chromium Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+import("//brave/build/cargo.gni")
```

Comment:
I'd like to avoid the dependency back on `//brave` here, but unfortunately I can't think of a good way to handle it right now so maybe we can come back to it later when there's more time

Reply:
👍